qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -44,6 +45,11 @@
         gagent_host_path = "/tmp/${qemu_ga_pkg}"
         gagent_download_cmd = "wget https://fedorapeople.org/groups/virt/virtio-win/direct-downloads/latest-qemu-ga/${qemu_ga_pkg} -O ${gagent_host_path}"
         gagent_guest_dir = "C:\qemu-ga"
+        src_qgarpm_path= "ls /tmp/qemu-ga-win*.rpm"
+        transfer_host_path = "/tmp/"
+        download_qgarpm = "wget -qP ${transfer_host_path} %s"
+        installrpm_getmsi = "rpm -Uvh --nodeps --force %s > /dev/null"
+        copy_qgamsi = "cp -r /usr/i686-w64-mingw32/sys-root/mingw/bin/qemu-ga-i386.msi /usr/x86_64-w64-mingw32/sys-root/mingw/bin/qemu-ga-x86_64.msi ${transfer_host_path}"
```

Comment:
the two paths are not same, one is '/usr/x86_64-w64-mingw32' and the other one is '/usr/i686-w64-mingw32' and both of them are from running command 'rpm -ql'

Reply:
@6-dehan I mean, that should not be static, it better be dynamically queried from `rpm -ql`, in case it might changes or it has different location on other platform.

Reply:
okay, patch has been updated for this part

(1/1) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2022.x86_64.io-github-autotest-qemu.qemu_guest_agent.virtio_serial.gagent_install.q35: PASS (117.14 s)